aiida/tools/importexport/common/exceptions.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+###########################################################################
+# Copyright (c), The AiiDA team. All rights reserved.                     #
+# This file is part of the AiiDA code.                                    #
+#                                                                         #
+# The code is hosted on GitHub at https://github.com/aiidateam/aiida-core #
+# For further information on the license, see the LICENSE.txt file        #
+# For further information please visit http://www.aiida.net               #
+###########################################################################
+"""Module that defines the exceptions thrown by AiiDA's export/import module.
+
+Note: In order to not override the built-in `ImportError`, both `ImportError` and `ExportError` are prefixed with
+    `Aiida`.
+"""
+from __future__ import division
+from __future__ import absolute_import
+from __future__ import print_function
+
+from aiida.common.exceptions import AiidaException
+
+__all__ = ('ExportImportException', 'AiidaExportError', 'AiidaImportError', 'CorruptArchive',
+           'IncompatibleArchiveVersionError', 'ExportValidationError', 'ImportUniquenessError', 'ImportValidationError',
+           'ArchiveMigrationError', 'MigrationValidationError', 'DanglingLinkError')
+
+
+class ExportImportException(AiidaException):
+    """Base class for all AiiDA export/import module exceptions."""
+
+
+class AiidaExportError(ExportImportException):
```

Comment:
Suggestion: `AiidaExportError` -> `ArchiveExportError`. I don't really like the `Aiida` name in variables. Plus you already have `ArchiveMigrationError`, so this would line up nicely. But of course, just personal preference :)

Reply:
I agree, don't like using `Aiida` in variables either. I only used it here to not overwrite Python's internal `ImportError` and make it consistent between the overall/umbrella exception for import and export. But I can change both to `ArchiveImportError` and `ArchiveExportError`?

Reply:
that would be my choice yeah :+1: